### ISSUE TITLE
How to get only customUrl parameter instead of attachmentReports fallback ?

### ISSUE THREAD

**Issue by samixchoumi** [YOU54F/cypress-plugins#1216]
### cypress-slack-reporter plugin

Hello @YOU54F, do you know why when i set `--custom-url=$CI_PIPELINE_URL`, it add this boring `/cypress/reports/report.html` at the end of the link ? I verified (to be sure) that my `$CI_PIPELINE_URL` and it haven't it.
It's a bit annoying to have this because the link i've set is redirecting to a 404 instead of the good link.

EDIT : 
After digging in your code @YOU54F, i found it's because `attachmentReports` is what is sent as fallback. But how can i don't send this one ? I just want my `customUrl`, not use the `attachmentReports` url.

EDIT2 : 
Instead of returning : 
```
return {
      totalSuites: 0,
      totalTests: 0,
      totalPasses: 0,
      totalFailures: 0,
      totalDuration: 0,
      reportFile: [],
      status: "error",
    };
```
If their is no HTML `reportFile`, it could be an idea to only return `customUrl` and all report data from report.json as usually done. 
Or a better idea : allow a new optional parameter like "useOnlyCustomUrl" (default not used) that when it's call in the CLI, `customUrl` is the good URL to return instead of `attachmentReportsUrl`.
> Added this PR : https://github.com/YOU54F/cypress-plugins/pull/1219 to show what i'm talking about (but can't test it, i'm a beginner using GitHub and i don't know how to let it available on npm repository)

**Comment by samixchoumi** [comment]
Hello@YOU54F,
As you merged my PR, is it available on npm to try it ?

**Comment by YOU54F** [comment]
Hey, it's not, it failed the master run and won't build due to the updated dependencies and impact on imports, got part way through sorting it but got distracted, will probably give it another tickel at the weekend 👍🏾 

You can build the project and publish it as a fork on npm under your own scope by

1. changing the package.json to `@<npm username>/cypress-slack-reporter`
2. publish with `npm publish --access public`, you can add the `--dry-run` flag to test it out beforehand.

You will need to build the project so the `./dist` folder is available 👍🏾

**Comment by samixchoumi** [comment]
Hello, thanks for your help. I think i will wait for your next dependencies update, i will take a look too if i can help you a bit with that.
I don't have any `npm` account available and it seem that the build fail too because of some `yarn` error..

**Comment by samixchoumi** [comment]
Hello @YOU54F, just fixed dependencies from `package.json`, you can take a look here : **https://github.com/YOU54F/cypress-plugins/pull/1225**

**Comment by samixchoumi** [comment]
@YOU54F Hello, any news about this project ?

**Comment by samixchoumi** [comment]
@YOU54F Hello, any news about this project ?

**Comment by YOU54F** [comment]
Should be released now as `1.5.3`

**Comment by YOU54F** [comment]
> Thanks @YOU54F, it works fine now !

awesome, thanks for the feedback and change
